net-misc/feather/feather-1.0.1.ebuild

```
@@ -0,0 +1,107 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+#Note: this is like a tree, with dependencies-of-dependencies
+#You need to update all of these recursively every version bump.
+#But at least they are distfiles if github goes down. ¯\_(ツ)_/¯
+MONERO_DIST_COMIT="6a2b96394d3c81a4ccf9be0daea02afe5f6f3683"
+	MINIUPNP_DIST_COMIT="544e6fcc73c5ad9af48a8985c94f0f1d742ef2e0"
+	RANDOMX_DIST_COMIT="fe4324e8c0c035fec3affd6e4c49241c2e5b9955"
+	RAPIDJSON_DIST_COMIT="129d19ba7f496df5e33658527a7158c79b99c21c"
+	SUPERCOP_DIST_COMIT="633500ad8c8759995049ccd022107d1fa8a1bbc9"
+	TREZORCOMMON_DIST_COMIT="bff7fdfe436c727982cc553bdfb29a9021b423b0"
+	UNBOUND_DIST_COMIT="0f6c0579d66b65f86066e30e7876105ba2775ef4"
+
+DESCRIPTION="A free, open-source Monero wallet"
+HOMEPAGE="https://featherwallet.org"
+SRC_URI="https://github.com/feather-wallet/feather/archive/refs/tags/${PVR}.tar.gz -> ${PF}.tar.gz
```

Forgot to add, we should use ${PV} here. ${PVR} includes the ebuild revision (e.g. `feather-1.0.1-r1.ebuild`).

```suggestion
SRC_URI="https://github.com/feather-wallet/feather/archive/refs/tags/${PV}.tar.gz -> ${PF}.tar.gz
```

---

I've made the change, but why shouldn't we use $PVR? is the revision number just for the change on our side of the ebuild?

---

Yep. We increment the revision when changing the ebuild so that users rebuild it.

https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html